Docs/source/running_cpp/parameters.rst

```diff
@@ -1070,6 +1070,12 @@ Numerics and algorithms
 * ``warpx.mag_time_scheme_order`` (`1` or `2`; default: `1`)
     The value of the time advancement scheme of M field. `mag_time_scheme_order==1` is the 1st-order Eulerian scheme and `mag_time_scheme_order==2` is the 2nd-order trapezoidal scheme for the LLG equation. This requires `USE_LLG=TRUE` in the GNUMakefile.
 
+* ``warpx.mag_secondorder_normalization`` (`1` or `2`; default: `1`)
```

Comment:
I don't have a preference; as long as we know we are only going to call this for varying material properties (and thus use the macroscopic properties).  If you are ok with this I'll move it.  You could also argue the same for warpx.mag_time_scheme_order; what do you think of that one?

Reply:
yes.. all mag_ input variables can be moved to a difference class, instead of WarpX.
I will merge this for now and crate an issue for this to be done.

Reply:
thumbs up emoji